### PR TITLE
Livebooks 1.0

### PR DESCRIPTION
**Pull request by bartkrak**
solves: https://github.com/membraneframework/membrane_core/issues/704

### CODE REVIEW

File: livebooks/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+This folder contains interactive livebook examples. To launch them you need to install livebook first.
+
+## Installation
+
+1. Install Livebook
+
+    It is recommended to install Livebook via command line ([see official installation guide](https://github.com/livebook-dev/livebook#escript)). 
+
+    If livebook was installed directly from the official page, one should add `$PATH` variable to the Livebook environment:
+    ![Setting path](./assets/path_set.png "Title")
```
Comment (FelonEkonom) [inline livebooks/README.md:12]: Was this in `guide`? `1.` in line 7 looks little bit weird

File: livebooks/audio_mixer/audio_mixer.livemd [hunk 2]
```diff
@@ -111,10 +111,10 @@ structure = beeps_split ++ [beep_audio_input, background_audio_input, mixer_outp
 ## Playing audio
 
 ```elixir
-alias Membrane.RemoteControlled, as: RC
+alias Membrane.RCPipeline, as: RC
 
-pipeline = RC.Pipeline.start!()
-RC.Pipeline.exec_actions(pipeline, spec: structure, playback: :playing)
+pipeline = RC.start!()
+RC.exec_actions(pipeline, spec: structure)
```
Comment (FelonEkonom) [inline livebooks/audio_mixer/audio_mixer.livemd:117]:
```suggestion
RC.exec_actions(pipeline, spec: spec)
```

File: livebooks/messages_source_and_sink/messages_source_and_sink.livemd [hunk 3]
```diff
@@ -18,7 +18,7 @@ defmodule MessageSource do
   require Membrane.Logger
 
   def_output_pad(:output,
-    mode: :push,
+    flow_control: :push,
     accepted_format: _any
   )
```
Comment (FelonEkonom) [inline livebooks/messages_source_and_sink/messages_source_and_sink.livemd:24]: Remove the brackets from the pad definitions

File: livebooks/rtmp/rtmp_receiver.livemd [hunk 4]
```diff
@@ -54,16 +53,17 @@ defmodule RTMP.Receiver.Pipeline do
...
+        output_stream_structure: :annexb
       })
       |> via_in(:video)
       |> get_child(:player)
 
     player = child(:player, %Membrane.Kino.Player.Sink{kino: kino})
 
     structure = [source, playing_audio, playing_video, player]
-    {[spec: structure, playback: :playing], %{}}
+    {[spec: structure], %{}}
```
Comment (FelonEkonom) [inline livebooks/rtmp/rtmp_receiver.livemd:66]:
```suggestion
    {[spec: spec], %{}}
```
You can rename `structure` to `spec` in similar places as well

File: livebooks/rtmp/rtmp_sender.livemd [hunk 5]
```diff
@@ -40,22 +40,20 @@ defmodule RTMP.Sender.Pipeline do
...
-      |> child(:video_payloader, Membrane.MP4.Payloader.H264)
+      |> child(:video_payloader, %Membrane.H264.Parser{output_stream_structure: :avc1})
 
     audio_source =
       child(:audio_source, %Membrane.Hackney.Source{
         location: @audio_url,
         hackney_opts: [follow_redirect: true]
       })
       |> child(:audio_parser, %Membrane.AAC.Parser{
-        in_encapsulation: :ADTS,
```
Comment (FelonEkonom) [inline livebooks/rtmp/rtmp_sender.livemd:58]: I am not a specialist, when it comes to AAC, but make sure, that it is safe, to remove this option

File: livebooks/messages_source_and_sink/messages_source_and_sink.livemd (hunk 3 above)
Comment (bartkrak) [reply livebooks/messages_source_and_sink/messages_source_and_sink.livemd:24]: I did it, but Livebook does some kind of auto-formatting and adds these brackets every time you just open the file in Livebook.

File: livebooks/rtmp/rtmp_sender.livemd (hunk 5 above)
Comment (bartkrak) [reply livebooks/rtmp/rtmp_sender.livemd:58]: In current version of acc parser this field no longer exists. Documentation says, correct acc stream format is supplied with the stream. And it works just fine

**Review by FelonEkonom — APPROVED** [review]
Branch `livebooks-1.0` can be merged into `master`

**Pull request merged**